README.md

```diff
@@ -30,6 +28,55 @@ module "network" {
 
 ```
 
+Example adding a network security rule for SSH:
+-----------------------------------------------
+
+```hcl
+variable "resource_group_name" { }
+
+module "network" {
+  source              = "../terraform-azurerm-network"
+  resource_group_name = "${var.resource_group_name}"
+  location            = "westus"
+  address_space       = "10.0.0.0/16"
+  subnet_prefixes     = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
+  subnet_names        = ["subnet1", "subnet2", "subnet3"]
```

Comment:
Can you make this example only one or two subnets and include them in the script ? Also please move the subnets ahead of Network module. This will be better for readability.